src/chainparams.cpp

```diff
@@ -200,7 +200,11 @@ class CCustomParams : public CChainParams {
 
         genesis = CreateGenesisBlock(consensus, strNetworkID, 1296688602, genesisChallengeScript, 1);
         if (initialFreeCoins != 0) {
-            AppendInitialIssuance(genesis, COutPoint(uint256(commit), 0), parentGenesisBlockHash, 1, initialFreeCoins, 0, 0, CScript() << OP_TRUE);
+	    if (initialReissuanceTokens != 0) {
```

Comment:
I don't think these checks should be nested? Each argument should individually be applied.

Reply:
Just thinking - if initialFreeCoins is 0 you can't issue any subsequent assets or reissue as you have nothing to fund the tx to create them and I think you end up with a frozen chain (unless you adjust fee settings to zero I guess). So now you mention it having ifc = 0 itself seems like it could cause issues. Hmmm. 

But yes, I can split out and will do. Thanks.

Reply:
You could always set the reissuance tokens as fee asset :)

Reply:
ah ha, will try that later, didn't think you could do that!

I'll split over two commits, add a test, change the nested if and snake_case cheers.

Reply:
I figure the more flexible, the less we have to change later.